tutorials/3d/procedural_geometry/arraymesh.rst

```diff
@@ -233,6 +299,74 @@ that you find online.
 
       # Insert committing to the ArrayMesh here.
 
+ .. code-tab:: csharp C#
+
+    public partial class ArrayMeshInstance : MeshInstance3D
+    {
+        public override void _Ready()
+        {
+            // Insert setting up the surface array and lists here.
+
+            var rings = 50;
```

Comment:
If we want to match GDScript, those should be fields in the class instead of local variables.

Reply:
Yes, good catch. Updated.